tests/cases/conformance/jsdoc/jsdocParseHigherOrderFunction.ts

```diff
@@ -0,0 +1,7 @@
+// @noemit: true
+// @allowjs: true
+// @checkjs: true
+// @strict: true
+// @Filename: paren.js
+/** @type {function((string), function((string)): string): string} */
+var x = s => s.toString()
```

Comment:
Would be nice to give the arrow function 2 parameters to verify that the contextual type is correct.

Reply:
Oops, I forgot to copy over the newer version of my in-editor test buffer. Fixed.